source/application/AMPIC887DriverServer/GCS2Commands/AMGCS2GetCurrentPositionCommand.h

```diff
@@ -30,13 +31,8 @@ class AMGCS2GetCurrentPositionCommand : public AMGCS2Command
 	  * the last time the command was run. If the command has not been run, or was
 	  * not successful when last run, then an empty map is returned.
 	  */
-	QHash<AMGCS2::Axis, double> axisPositions();
+	AMPIC887AxisMap<double> axisPositions();
```

Comment:
If you're expecting to call this really often, you could probably make this a `const AMPIC887AxisMap<double> &axisPositions() const;` method since `axisPositions_` is a member of this class.  That way you don't make copies every time you call it.  If this class isn't long lived then it's not a good idea, but just thought I'd throw it out there.